manifests/julia/cert-manager-issuers.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  namespace: cert-manager
+  name: letsencrypt-staging
+spec:
+  acme:
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: me@juliaogris.com
```

Comment:
haven't seen you use that address before. hello you!

Reply:
🌊